editor.planx.uk/src/pages/Login.tsx

```diff
@@ -35,16 +43,21 @@ const Login: React.FC = () => {
           Plan✕
           <PhaseText>beta</PhaseText>
         </Typography>
-        <Link
-          variant="body1"
-          color="#FFFFFF"
+        <LoginButton
+          variant="contained"
+          color="secondary"
           href={`${
             process.env.REACT_APP_GOOGLE_OAUTH_OVERRIDE ??
             process.env.REACT_APP_API_URL
           }/auth/google`}
         >
-          Login with Google
-        </Link>
+          <GoogleIcon />
+          Continue with Google
+        </LoginButton>
+        <LoginButton variant="contained" color="secondary" href={`...`}>
+          <MicrosoftIcon />
+          Continue with Microsoft
+        </LoginButton>
```

Comment:
Shall we simply `disable` this button and adjust text to read something like `Continue with Microsoft (Coming soon)` and get this merged to `main` ahead of demo tomorrow & for future implementation work to rebase on top of? Might be easier than keeping an open PR around!

Reply:
Great shout, I've got a commit coming in for this.

Reply:
Just came here to suggest this very thing! I'll then rebase #3221 for the demo tomorrow.